### PR TITLE
ReadMe Updated

### PR DESCRIPTION
**Pull request by Prakhar-commits**
Updated the Readme for College-Predictor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,44 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+## About
 
-## Getting Started
+College Predictor provides the list of colleges given :
```
Comment (suryabulusu) [inline README.md:3]: `provides a list of colleges with opening and closing ranks given:`

File: README.md [hunk 2]
```diff
@@ -1,36 +1,44 @@
...
+
+- Exam
+- Category
+- Category rank
+- Round Number
+- Gender
+- Home State
 
-First, install dependencies (`nextjs-course`) and run the development server:
+It also finds the Scholarships given :
```
Comment (suryabulusu) [inline README.md:12]: `finds a list of applicable scholarships given:`

File: README.md [hunk 3]
```diff
@@ -1,36 +1,44 @@
...
-To learn more about Next.js, take a look at the following resources:
-
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+## Deploy
 
-## Deploy on Vercel
+- Using AWS amplify
```
Comment (suryabulusu) [inline README.md:40]: Amplify *

**Review by suryabulusu — COMMENTED** [review]
left comments

File: README.md (hunk 1 above)
Comment (Prakhar-commits) [reply README.md:3]: updated

File: README.md (hunk 2 above)
Comment (Prakhar-commits) [reply README.md:12]: resolved

File: README.md (hunk 3 above)
Comment (Prakhar-commits) [reply README.md:40]: updated

**Pull request merged**